pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
I think for the Dockerfile we're fine just using the latest. The client is backward compat and parameterizing the Dockerfile isn't always super straight forward.

For the `KubernetesHyperkubeImagePrefixName`, I'd suggest leaving off the colon, or better yet, having a map of "version" -> "list of versioned images" where you can just lookup instead of having to string manipulate. Also, this will help if they ever renamed the images (which could happen, they didn't use to have the -ARCH suffix).

Also, there is no such thing as `kubernetes-dashboard` version `1.5.3`. It doesn't exist. The dashboard doesn't always release with K8s itself. So yes, for "k8s 1.5.3", the dashboard image should still be 1.5.1.